build.assets/tooling/cmd/changelog/prparser_test.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	_ "embed"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//go:embed testdata/expectedCL.md
```

Comment:
I don't think we need to use embedding for tests. The local filesystem is always going to be available, so we don't need to embed the data in the binary.

Reply:
Yeah you're right I did embedding hoping it would be easier but it's actually clearer to just read from file so I updated it.